### PR TITLE
Item Controller Route

### PR DESCRIPTION
**Pull request by chriznm**
New route and controller for Item(s)

### CODE REVIEW

**Comment by chriznm** [comment]
No MB yet, sorry.

File: src/api-docs/swagger.json [hunk 1]
```diff
@@ -129,7 +133,7 @@
                   "description":"OK",
                   "schema":{
                      "type": "array",
-                     "items": {
+                     "item": {
```
Comment (andreschabba) [inline src/api-docs/swagger.json]: I think that's a syntaxis of swagger, so the correct approach is items.. Could you confirm?

File: src/api-docs/swagger.json [hunk 2]
```diff
@@ -256,7 +260,7 @@
...
```
Comment (andreschabba) [inline src/api-docs/swagger.json]: same here

File: src/api-docs/swagger.json [hunk 3]
```diff
@@ -288,6 +292,133 @@
...
+                  "required":true,
+                  "type":"integer",
+                  "format":"int64"
+               }
+            ],
+            "responses":{
+               "200":{
+                  "description":"OK",
+                  "schema":{
+                     "$ref":"#/definitions/ItemType"
```
Comment (andreschabba) [inline src/api-docs/swagger.json]: Why is returning here an ItemType?

File: src/api-docs/swagger.json [hunk 4]
```diff
@@ -288,6 +292,133 @@
...
+            "tags":[
+               "Item"
+            ],
+            "summary":"Get all item in system",
+            "responses":{
+               "200":{
+                  "description":"OK",
+                  "schema":{
+                     "type": "array",
+                     "item": {
```
Comment (andreschabba) [inline src/api-docs/swagger.json]: also here, (items)

File: src/api-docs/swagger.json [hunk 5]
```diff
@@ -383,7 +514,7 @@
                   "description":"OK",
                   "schema":{
                      "type": "array",
-                     "items": {
+                     "item": {
```
Comment (andreschabba) [inline src/api-docs/swagger.json]: items*

File: src/routes/itemRoutes.js [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+    app.route('/items')
+      .post(itemAddNew)
+      .get(itemGetAll);
+    app.route('/item/:itemId')
+      .get(itemGetById)
+      .put(itemUpdateById)
+      .delete(itemRemoveById);
+  };
+  
+  module.exports = itemRoutes;
```
Comment (andreschabba) [inline src/routes/itemRoutes.js]: Remember that always we should leave an empty line at the end of each file.

**Review by andreschabba — APPROVED** [review]

**Pull request merged**